tests/unit/CorrectNameRequest.spec.ts

```diff
@@ -309,12 +309,13 @@ describe('CorrectNameRequest', () => {
             }],
             nrNum: 'NR 1234567',
             requestTypeCd: 'BC',
-            request_action_cd: 'CNV',
+            request_action_cd: 'NEW',
             entity_type_cd: 'BC',
             applicants: {
               phoneNumber: '250 516 8257',
               emailAddress: 'mock@example.com'
-            }
+            },
+            legalType: 'GP'
```

Comment:
These unit test changes now only validate a GP/NEW NR and no longer validate a BEN CNV NR. Please add a unit test (or parameterize it) instead.